### PR TITLE
[RW-6989][risk=no] UI: Fix texts string in Delete Workspace Error modal 

### PR DESCRIPTION
**Pull request by aweng98**
Error message screenshot:

Screen Shot 2021-07-01 at 5 06 56 PM

Steps I used to force error modal to display:
* Start local UI server.
* Connect to local UI server on 2 different incognito pages.
* Delete a workspace in page 1, then quickly delete the same workspace again in page 2.
* Because two delete calls were made, delete in page 2 failed when delete in page 1 succeeded.

New look after change:

Screen Shot 2021-07-01 at 5 03 23 PM

After close modal, page is not auto reloaded:

Screen Shot 2021-07-01 at 5 03 34 PM

User reload the page. workspace is gone.

Screen Shot 2021-07-01 at 5 03 50 PM

### CODE REVIEW

File: ui/src/app/pages/workspace/workspace-card.tsx [hunk 1]
```diff
@@ -162,19 +162,18 @@ export class WorkspaceCard extends React.Component<WorkspaceCardProps, Workspace
...
-    const {workspace} = this.props;
-    this.setState({
-      confirmDeleting: false,
-      loadingData: true});
-    try {
-      await workspacesApi().deleteWorkspace(workspace.namespace, workspace.id);
-      this.setState({loadingData: false});
-      await this.props.reload();
-    } catch (e) {
-      this.setState({bugReportOpen: true, bugReportError: 'Could not delete workspace', loadingData: false});
```
Comment (calbach) [inline ui/src/app/pages/workspace/workspace-card.tsx:175]: This appears to have been the only usage of these two pieces of state: `bugReportOpen`, `bugReportError`, please remove them and any related code from this class

**Review by calbach — COMMENTED** [review]
Nice, thanks for the cleanup

File: ui/src/app/pages/workspace/workspace-card.tsx (hunk 1 above)
Comment (aweng98) [reply ui/src/app/pages/workspace/workspace-card.tsx:175]: Good catch. Removed.

**Review by calbach — APPROVED** [review]
Looks good once presubmit is passing (needs lint)

File: ui/src/app/pages/workspace/workspace-card.tsx [hunk 2]
```diff
@@ -153,28 +150,25 @@ export class WorkspaceCard extends React.Component<WorkspaceCardProps, Workspace
...
   }
 
-  async deleteWorkspace() {
-    const {workspace} = this.props;
-    this.setState({
-      confirmDeleting: false,
-      loadingData: true});
-    try {
-      await workspacesApi().deleteWorkspace(workspace.namespace, workspace.id);
-      this.setState({loadingData: false});
```
Comment (jmthibault79) [inline ui/src/app/pages/workspace/workspace-card.tsx:172]: looks like this state is no longer set.  Does this mean that there is no longer a spinner here?  @aweng98

Comment (aweng98) [reply ui/src/app/pages/workspace/workspace-card.tsx:172]: I saw spinner during my local testing. Did you think this is wrong or see some weird behavior?

Comment (jmthibault79) [reply ui/src/app/pages/workspace/workspace-card.tsx:172]: No, everything looks fine.  We can actually remove this now: https://github.com/all-of-us/workbench/pull/5218

**Pull request merged**